### DISCUSSION TITLE
Upgrade path from 2.5.6

### DISCUSSION THREAD

**Discussion by brosky** [netbox-community/netbox#5802] [Q&A] [answered]
Hi, 

I use a 2.5.6 instance and I want to upgrade however whatever method I try , no luck. (python dependencies, centos 7.6)

What's the easiest way to get a uptodate version ? Can I start with a clean fresh install and import the DB ?

**Comment by candlerb** [comment, marked as answer]
> Can I start with a clean fresh install and import the DB ?

Yes you can: https://netbox.readthedocs.io/en/stable/administration/replicating-netbox/

After importing the DB to the new server, run the `./upgrade.sh` script which will apply all the migrations (database changes) required to bring it up to date.

**Reply by brosky** [reply]
Well, I did that:

new ubuntu20.04 vm, installed netbox from scratch, created superuser, tested everything is ok. 
prod server, dumped the db, copied to new server
dropped db, import new db, run ./upgrade.sh
create new superuser (since none is recognized, new or old) started in venv and app works but no data. 

What i'm doing wrong ? :)

**Reply by candlerb** [reply]
Probably you restored to the wrong database or schema - maybe you restored to the `postgres` database instead of the `netbox` one.

Try a simple query, e.g. `select * from dcim_device` after you've restored, to be sure.

There is no need to create new superuser.  If the database is restored correctly, then it will include all users including the superuser(s).

**Reply by brosky** [reply]
> Probably you restored to the wrong database or schema - maybe you restored to the `postgres` database instead of the `netbox` one.

That was it. I did the import again and now I'm on the latest and greatest. 

Thank you!
